assets/scss/base/forms.scss

```diff
@@ -0,0 +1,27 @@
+.form-control {
+  background: rgba(0, 0, 0, 0.1);
+  border: 1px solid $aluminum;
+  border-radius: 5px;
+  display: block;
+  margin: auto;
+  padding: 2rem;
+  width: 35rem;
+}
+
+.button  {
+  background: rgba(0, 0, 0, 0.1);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.1)` should only be used in variable declarations; they should be referred to via variable everywhere else.